framework/widgets/LinkPager.php

```diff
@@ -42,6 +42,11 @@ class LinkPager extends Widget
      */
     public $options = ['class' => 'pagination'];
     /**
+     * @var array HTML attributes which will be applied to all link wrappers
+     * @since 2.0.13
+     */
+    public $linkWrapOptions = [];
```

Comment:
using the name $linkContainerOptions would be more consistent throughout the framework